src/main/java/org/opentripplanner/routing/algorithm/AStar.java

```diff
@@ -270,19 +274,8 @@ void runSearch(long abortTime) {
                 runState.foundPathWeight = runState.u.getWeight();
                 runState.options.rctx.debugOutput.foundPath();
                 // new GraphPath(runState.u, false).dump();
-                /* Only find one path at a time in long distance mode. */
-                if (runState.options.longDistance) {
-                    break;
-                }
-                /* Break out of the search if we've found the requested number of paths. */
-                if (runState.targetAcceptedStates.size() >= runState.options.getNumItineraries()) {
```

Comment:
Are we sure, that we want to remove this part as well? (this is not associated with the longDistanse property)

Reply:
Right now it is a dead code - we are always in a long distance mode, so we always break from this loop on previous condition. When I replaced 
```
if (runState.options.longDistance) {
  break;
}
```
with `break;`, this part became unreachable. Maybe we could think of some optimization which would take advantage of continuing the same search instead of starting a new one, but it is a long discussion.